src/applications/ivc-champva/10-7959C/chapters/healthInsuranceInformation.js

```diff
@@ -12,7 +12,7 @@ import {
   yesNoUI,
   yesNoSchema,
 } from 'platform/forms-system/src/js/web-component-patterns';
-import { nameWording } from '../helpers/utilities';
+import { nameWording, nameWordingExt } from '../helpers/utilities';
```

Comment:
`nameWordingExt` is a wrapper around multiple `nameWording` calls to give us some commonly used combinations rather than duplicating a bunch of ternaries.